src/code_f2470_len_27f0.c

```diff
@@ -191,26 +191,150 @@ INCLUDE_API_ASM(code_f2470_len_27f0, SetNpcFlagBits);
 
 INCLUDE_API_ASM(code_f2470_len_27f0, GetNpcPos);
 
-INCLUDE_API_ASM(code_f2470_len_27f0, EnableNpcShadow);
+ApiStatus EnableNpcShadow(ScriptInstance* script, s32 isInitialCall) {
+    Bytecode* ptrReadPos = script->ptrReadPos;
+    NpcId npcID = get_variable(script, *ptrReadPos++);
+    s32 enableShadow = get_variable(script, *ptrReadPos++);
+    Npc* npcPtr = resolve_npc(script, npcID);
```

Comment:
I don't think we should name temps thingPtr generally, this and all below can just be `npc`